docs/README.md

```diff
@@ -40,6 +40,7 @@ The documentation is divided into the following sections:
 
 | ACA   | ADF           | ACS       | Node | Angular |
 |-------|---------------|-----------|------|---------|
+| 5.2.x | 7.0.0-alpha.6 | 23.3      | 18.x | 16.x    |
```

Comment:
We released ACS 23.4 now so this will be compatible version

Reply:
Updated